src/helpers/helpers.core.js

```diff
@@ -308,6 +308,19 @@ var helpers = {
 
 		ChartElement.__super__ = me.prototype;
 		return ChartElement;
+	},
+
+	/**
+	 * If argument is string, return it. If it is an array, join its elements using
+	 * a space string (' ').
+	 */
+	joinIfArray: function(stringOrArray) {
```

Comment:
We generally try to avoid introducing new public APIs. You should probably either prefix it with `_` and add a `@private` jsdoc tag or remove this method and inline the code the one place it's called

Reply:
77f932c resolves this